src/pybind/pynmodl.cpp

```diff
@@ -138,12 +135,13 @@ void init_visitor_module(py::module& m);
 void init_ast_module(py::module& m);
 void init_symtab_module(py::module& m);
 
-
 PYBIND11_MODULE(_nmodl, m_nmodl) {
     m_nmodl.doc() = "NMODL : Source-to-Source Code Generation Framework";
     m_nmodl.attr("__version__") = nmodl::Version::NMODL_VERSION;
 
-    py::class_<nmodl::PyNmodlDriver> nmodl_driver(m_nmodl, "NmodlDriver", nmodl::docstring::driver);
+    py::class_<nmodl::parser::NmodlDriver>(m_nmodl, "nmodl::parser::NmodlDriver");
+    py::class_<nmodl::PyNmodlDriver, nmodl::parser::NmodlDriver> nmodl_driver(
+        m_nmodl, "NmodlDriver", nmodl::docstring::driver);
```

Comment:
Is there a simple "why"?

Reply:
https://stackoverflow.com/questions/62854830/error-by-wrapping-c-abstract-class-with-pybind11

Reply:
Thanks!